rest-api-project/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const cors = require("cors");
+const zines = require("./zines.js");
+
+const app = express();
+
+app.use(cors());
+
+const PORT = 5001; 
```

Comment:
In general, it's considered best practice to declare most of your constants at the top of a particular scope before you write any application code. Let's say you get down to line 32 and forget what `PORT` was set to. Where are you naturally going to go to look up the assignment? Put your constant declaration there.